### PR TITLE
Add singleton class instructions to porting guide

### PR DESCRIPTION
**Pull request by Vatyx**

### CODE REVIEW

**Comment by girayimprobable** [comment]
Streaming query?

**Comment by Vatyx** [comment]
O jeez

File: docs/content/porting-unreal-project-to-gdk.md [hunk 1]
```diff
@@ -160,6 +160,14 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
         [InteropCodeGen.Settings]
         OutputPath=YourProject\Generated\
         ```
+        
+    1. In `SnapshotGenerator.SingletonActorClasses`, modify the `StarterProjectGameStateBase` to the GameState class in your game.
```
Comment (anne-edwards) [inline docs/content/porting-unreal-project-to-gdk.md]: Should GameState be `GameState` (as it's a class)?

File: docs/content/porting-unreal-project-to-gdk.md [hunk 2]
```diff
@@ -160,23 +160,50 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
...
 
         ;YourProjectCharacter
-        YourProjectCharacter=YourProjectCharacter.h
-        YourProjectCharacter=OtherDependency.h
+        /Script/YourProject.YourProjectCharacter=YourProjectCharacter.h
+        /Script/YourProject.YourProjectCharacter=OtherDependency.h
         ```
-
+        
+1. Open your UnrealWorker worker config located in `<ProjectRoot>\spatial\workers\unreal\spatialos.UnrealWorker.worker.json`. Edit the  streaming queries to be the generated component for your Singletons. For the example above your streaming queries would look like so.
```
[the remark this answers is not in the document]
Comment (m-samiec) [reply docs/content/porting-unreal-project-to-gdk.md]: Perhaps this should be, `Add your new Singleton generated component to the streaming queries. For instance:`

File: docs/content/porting-unreal-project-to-gdk.md [hunk 3]
```diff
@@ -160,23 +160,50 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
...
+    
+For example:
```
Comment (m-samiec) [inline docs/content/porting-unreal-project-to-gdk.md]: I'd remove this as you're already said this is an example.

**Pull request merged**